pyproject.toml

```diff
@@ -141,3 +141,11 @@ target-version = "py311"
 
 [tool.setuptools]
 py-modules = []
+
+[tool.coverage.run]
+plugins = [
+    'django_coverage_plugin',
```

Comment:
Do we need to add this dependency to our project dependencies?

Reply:
Yes, install is available with pip.
I didn't want to tamper with the requirements.txt that's why I didn't include it

Reply:
Go ahead and add it to the pyproject.toml requirements list and run `make compile-deps`, then commit the changes to requirements.txt files(s). It's a good improvement.

Also, make sure to pull the recent change from GitHub where I removed the duplicate `[run]` definition that was breaking CI.

Reply:
Alternatively, remove the changes related to `django_coverage_plugin` and take that up in a separate task: https://github.com/WesternFriend/WF-website/issues/663

Reply:
I just did that